### PR TITLE
Pear info <channelName>

### PR DESCRIPTION
**Pull request by ruy-dan**
Updates pear info command with the following:

pear info [key]  -> pear info [key|channel]
pear info [key|channel] --changelog / pear info [key|channel] --no-changelog
pear info [key|channel] --full-changelog
pear info [key|channel] --metadata / pear info [key|channel] --no-metadata
pear info [key|channel] --key

### CODE REVIEW

File: cmd/usage.js [hunk 1]
```diff
@@ -69,17 +69,23 @@ module.exports = ({ fork, length, key }) => {
...
 
     ${infoBrief}
 
-    Supply a key to view application info
+    Supply a key or channel to view application info
 
     Without a key pear info shows Pear info
 
-    --json          Newline delimited JSON output
+    --json            Newline delimited JSON output
```
Comment (davidmarkclements) [inline cmd/usage.js:82]: but the least important things near the end, put --json last

File: lib/sidecar.js [hunk 2]
```diff
@@ -987,49 +987,73 @@ class Engine extends ReadyResource {
     await bundle.finalizeTracing()
   }
 
-  async * info ({ key } = {}, client) {
+  async * info ({ key, channel, dir, display } = {}, client) {
```
Comment (davidmarkclements) [inline lib/sidecar.js:990]: no to `display`, just match flags to keys as we do elsewhere - no nesting

File: cmd/usage.js (hunk 1 above)
Comment (ruy-dan) [reply cmd/usage.js]: moved json to the end

File: lib/sidecar.js (hunk 2 above)
Comment (ruy-dan) [reply lib/sidecar.js]: removed `display` with nested props

**Pull request merged**